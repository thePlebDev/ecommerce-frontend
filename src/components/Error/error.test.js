import React from 'react';
import { shallow } from 'enzyme';

import Error from './index.js';


test('it should render',()=>{
  const wrapper = shallow(<Error />)

  expect(wrapper).toMatchSnapshot();
})
