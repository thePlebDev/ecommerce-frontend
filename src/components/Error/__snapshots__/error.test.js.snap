// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it should render 1`] = `ShallowWrapper {}`;
